pkg/parser/sysl.wbnf

```diff
@@ -38,7 +38,13 @@ type_decl -> table | facade | alias | union;
 
 table -> mode=("!table"|"!type") NAME attribs?  ":" COMMENT*
      %!Indented(COMMENT_NO_NL | table_row COMMENT_NO_NL?) {
-        table_row -> (NAME array_size? "<:") type=(collection_type|type_spec) optional="?"? attribs? (":" annotations)?;
+       table_row -> COMMENT_NO_NL
+                    | table_field
+                    | annotation
+                    | table
+                    | SHORTCUT
+                    | table_def;
```

Comment:
what is a `table_def`? I don't see it anywhere in the grammar.

Reply:
that was a bad mistake. Meant to add table.

Reply:
replaced `table_def` to `table`